charts/openfga/values.yaml

```diff
@@ -36,36 +36,45 @@ service:
   type: ClusterIP
   port: 8080
 
-metrics:
-  ## @param metrics.enabled Expose OpenFGA metrics
-  ##
-  enabled: true
+telemetry:
+  trace:
+    enabled: false
+    otlp:
+      endpoint:
+    sampleRatio:
+
+  metrics:
+    ## @param telemetry.metrics.enabled enable/disable prometheus metrics on the '/metrics' endpoint
+    ##
+    enabled: true
 
-  ## @param metrics.port The port the '/metrics' endpoint is served on
-  ##
-  port: 2112
+    ## @param telemetry.metrics.addr the host:port address to serve the Metrics server on
+    addr: 0.0.0.0:2112
 
-  ## @param metrics.podAnnotations [object] Annotations for the Prometheus metrics on etcd pods
-  ##
-  podAnnotations:
-    prometheus.io/scrape: "true"
-    prometheus.io/port: "{{ .Values.containerPorts.prometheus }}"
+    ## @param telemetry.metrics.enableRPCHistograms enables prometheus histogram metrics for RPC latency distributions
+    enableRPCHistograms:
```

Comment:
Purposefully omit (here and elsewhere) values that should be defaulted to the server default values (e.g. no env variable will be set for it).